### ISSUE TITLE
Authorizations in Operate

### ISSUE THREAD

**Issue by meyerdan** [camunda-cloud/issues#14]
**Is your feature request related to a problem? Please describe.**
As Operate is deployed in larger organizations it becomes necessary to control access. There is two dimensions:
* _Role-based access_: read-only vs. ability to execute operations, ability to see token state only or also payload.
* _Resource-based access_: ability to access related to certain workflows only

**Describe the solution you'd like**
* For Role-based access: Operate should define application roles (read_only_operator, ...) that is supports. In [camunda/iam](https://github.com/camunda/iam) it should then be possible to assign an identity (user or group) such roles (ie. "Assign _read_only_operator_ to the group _ops_team_).
*For resource-based access: it needs to be possible to authorize access to certain processes based on the _bpmn process id_. 

**Your use case**
Automating processes in large organisations, heavily regulated industries

**Comment by meyerdan** [comment]
@menski do you consider this request to be in the scope of the product?

**Comment by menski** [comment]
Yes I see Authorizations in the scope of Operate. Regarding the following point I'm not sure if this UI/API should be in Operate, I would expect it in the IAM solution used.

> There should be both a UI and an API for creating such authorizations
